src/Tribe/Admin/Notice/Install_Event_Tickets.php

```diff
@@ -0,0 +1,436 @@
+	 */
+	public function is_active( $slug = '' ): bool {
+		return is_plugin_active( $this->get_plugin_path( $slug ) );
+	}
+
+	/**
+	 * Check if we're on the classic "Install Plugin" page.
+	 *
+	 * @since TBD
+	 *
+	 * @return boolean
+	 */
+	public function is_install_plugin_page(): bool {
+		return 'install-plugin' === tribe_get_request_var( 'action' );
+	}
+
+	/**
+	 * Should the `Install` notice be displayed?
+	 *
+	 * @since TBD
+	 *
+	 * @return bool True if the install notice should be displayed.
+	 */
+	public function should_display_notice_install() {
+		return ! $this->is_installed()
+			&& empty( tribe_get_request_var( 'welcome-message-the-events-calendar' ) )
+			&& ! $this->is_install_plugin_page();
+	}
+
+	/**
+	 * Should the `Activate` notice be displayed?
+	 *
+	 * @since TBD
+	 *
+	 * @return bool True if the activate notice should be displayed.
+	 */
+	public function should_display_notice_activate() {
+		return $this->is_installed() && ! $this->is_active() && ! $this->is_install_plugin_page();
+	}
+
+	/**
+	 * Install notice for `Event Tickets`.
+	 *
+	 * @since TBD
+	 */
+	public function notice_install() {
+		if ( ! $this->should_display_notice_install() ) {
+			return '';
+		}
+
+		$this->enqueue_assets();
+
+		$html = $this->get_template()->template(
+			'notices/install-event-tickets',
+			$this->get_template_data(),
+			false
+		);
+
+		tribe_notice(
+			'event-tickets-install',
+			$html,
+			[
+				'dismiss' => true,
+				'type'    => 'warning',
+			]
+		);
+	}
+
+	/**
+	 * Enqueue assets required for the notice.
+	 *
+	 * @since TBD
+	 *
+	 * @return void
+	 */
+	private function enqueue_assets() {
+		wp_enqueue_style( 'wp-components' );
+		tribe_asset_enqueue_group( self::$assets_group );
+	}
+
+	/**
+	 * Activate notice for `Event Tickets`.
+	 *
+	 * @since TBD
+	 */
+	public function notice_activate() {
+		if ( ! $this->should_display_notice_activate() ) {
+			return '';
+		}
+
+		$this->enqueue_assets();
+
+		$args = [
+			'description'  => __( 'You\'re almost there! Activate Event Tickets to manage attendee registration and ticket sales to your events, for free.', 'the-events-calendar' ),
+			'button_label' => __( 'Activate Event Tickets', 'the-events-calendar' ),
+			'action'       => 'activate',
+		];
+
+		$html = $this->get_template()->template(
+			'notices/install-event-tickets',
+			$this->get_template_data( $args ),
+			false
+		);
+
+		tribe_notice(
+			'event-tickets-activate',
+			$html,
+			[
+				'dismiss' => true,
+				'type'    => 'warning',
+			]
+		);
+	}
+
+	/**
+	 * Data for the notice template.
+	 *
+	 * @since TBD
+	 *
+	 * @param array $args Array with arguments to override the defaults.
+	 *
+	 * @return array The template args.
+	 */
+	private function get_template_data( $args = [] ): array {
+		$admin_url    = is_network_admin() ? network_admin_url( 'settings.php' ) : admin_url( 'admin.php' );
+		$redirect_url = add_query_arg( [ 'page' => 'tec-tickets-settings' ], $admin_url );
```

Comment:
The page's slug should come from the constant, not a literal string.

Reply:
This is required before ET is installed/active, so we don't have access to that constant at this point, it has to be hardcoded.

Reply:
Makes sense.